conda_forge_tick/auto_tick.py

```diff
@@ -225,7 +224,12 @@ def run(
                 )
             ]
 
-    if migrator.check_solvable and not is_recipe_solvable(feedstock_dir):
+    if (
+        migrator.check_solvable
+        or feedstock_ctx.attrs["conda-forge.yml"]
+        .get("bot", {})
+        .get("check_solvability", False)
```

Comment:
We use "check_solvable" in the migrator yaml. We should use that here.

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/pypy.yaml#L15

Reply:
We are that is on line 228. This is checking for if the *feedstock* wants to check solvability, regardless of the migrator (eg. the boto stack wants to check solvability for version bumps)

Reply:
I mean the key in the yaml. It should be named the same. I don’t want to have to remember which one goes where.